### ISSUE TITLE
`matplotlib.ticker.IndexFormatter` no longer available in matplotlib 3.5

### ISSUE THREAD

**Issue by titodalcanton** [mpld3/mplexporter#62]
I am having trouble using `mpld3` with matplotlib 3.5. The problem seems to be that `mplexporter` is using `matplotlib.ticker.IndexFormatter`, which has been deprecated for some time, and finally dropped in matplotlib 3.5:
```
>>> import matplotlib
>>> matplotlib.__version__
'3.5.0'
>>> from matplotlib.ticker import IndexFormatter
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: cannot import name 'IndexFormatter' from 'matplotlib.ticker' (/…/mpl-3.5-test/lib/python3.7/site-packages/matplotlib/ticker.py)
```

**Comment by titodalcanton** [comment]
https://github.com/matplotlib/matplotlib/issues/16631#issuecomment-594008777 might contain a useful suggestion for how to fix this.

**Comment by vladh** [comment]
Fixed in 0.5.6.
